python/rascaline-torch/tests/utils/cmake_prefix.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+import rascaline.torch
+
+
+def test_cmake_prefix():
+    assert os.path.exists(rascaline.torch.utils.cmake_prefix_path)
```

Comment:
@PicoCentauri here is the test checking that we can get the python variable, and the it points to something reasonable